ip/colormeter.py

```diff
@@ -2,7 +2,22 @@
 """Calculate color paramters"""
 
 import numpy as np
-
+import matplotlib.pyplot as plt
+
+
+def plot_hist(img):
+  hist, bins = np.histogram(img.flatten(), 256,[0, 256])
```

Comment:
Why is this 0-256? Should not be 1-256 or 0-255?

Reply:
https://matplotlib.org/3.1.0/api/_as_gen/matplotlib.pyplot.hist.html